packages/core-ethereum/src/index.ts

```diff
@@ -231,16 +231,24 @@ export default class HoprCoreEthereum extends EventEmitter {
 
   public async redeemAllTickets(): Promise<void> {
     if (this.redeemingAll) {
+      log('skipping redeemAllTickets because another operation is still in progress')
       return this.redeemingAll
     }
-    const _redeemAll = async () => {
+    this.redeemingAll = this.redeemAllTicketsInternalLoop()
+    return this.redeemingAll
+  }
+
+  private async redeemAllTicketsInternalLoop(): Promise<void> {
+    try {
       for (const ce of await this.db.getChannelsTo(this.publicKey.toAddress())) {
         await this.redeemTicketsInChannel(ce)
       }
-      this.redeemingAll = undefined
+    } catch (err) {
+      log(`error during redeeming all tickets`, err)
```

Comment:
shouldn't we throw here?

Reply:
If we throw we will have to catch it one layer up again, because otherwise we don't properly assign `this.redeemingAll`. I believe we should log this error and just keep retrying the future redeem operations again.